package.json

```diff
@@ -1,28 +1,28 @@
 {
   "name": "@icgc-argo/uikit",
   "version": "1.16.0",
+  "description": "ARGO UI component library",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "repository": {
     "type": "git",
     "url": "https://github.com/icgc-argo/uikit"
   },
   "scripts": {
-    "storybook": "NODE_PATH=./ start-storybook -s ./ -p 6006",
+    "storybook": "start-storybook -p 6006",
     "build-storybook": "build-storybook",
     "create-component": "plop --plopfile ./src/plopfile.js",
-    "gitmoji-config": "gitmoji -g",
-    "commit": "gitmoji -c",
     "type-check": "tsc -p ./tsconfig.json",
     "test": "NODE_PATH=./ ts-mocha -p ./tsconfig.test.json ./testSetup.tsx src/testSetup.tsx src/**/*.test.tsx",
     "test:watch": "NODE_PATH=./ ts-mocha --watch -p ./tsconfig.test.json ./testSetup.tsx src/testSetup.tsx src/**/*.test.tsx",
     "build::assets": "cp ./package.json ./dist/package.json && cp -r ./src/assets ./dist/assets && cp ./README.md ./dist/README.md",
     "build::ts": "tsc -p ./tsconfig.json",
-    "build::babel": "babel ./dist --out-dir ./dist",
-    "build::cleanup": "rm ./dist/**/*.jsx ./dist/*.jsx",
-    "build": "rm -rf ./dist && npm run build::ts && npm run build::babel && npm run build::assets && npm run build::cleanup",
-    "publish-uikit": "cd dist && npm publish --access public",
-    "prewatch": "rm -rf dist && mkdir dist && cd dist && npm link react react-dom",
-    "watch": "rm -rf ./dist && tsc-watch -p ./tsconfig.json --onSuccess \"npm run build::babel\"",
-    "postwatch": "rm -r dist"
+    "build::babel": "babel ./src --extensions .tsx,.ts,.js --out-dir ./dist",
+    "clean": "rm -rf dist && mkdir dist",
+    "declarations": "ttsc --emitDeclarationOnly",
+    "watch": "npm run clean && npm run build::assets && (npm run build::babel -- --watch --source-maps=inline & npm run declarations -- --watch)",
+    "build": "npm run clean && npm run build::assets && npm run declarations && npm run build::babel",
+    "publish-uikit": "cd dist && npm publish --access public"
   },
   "peerDependencies": {
     "react": "16.x.x || 17.x.x",
```

Comment:
any potential issues with this project being on 18 but requiring react 16 or 17 as a peer dependency?

Reply:
not with this specific dependency as it's a dev dependency (storybook needs it) but there will no doubt be some tweaks needed with peerDeps when we upgrade all the ARGO projects, so not worrying about it until then because not much I can debug in isolation. Not sure if all ARGO are on same react versions for instance.